tutorials/pubsub-spring-dedup-messages/pubsub-spring/pom.xml

```diff
@@ -15,12 +15,11 @@
   </parent>
 
   <properties>
-    <maven.compiler.source>1.8</maven.compiler.source>
-    <maven.compiler.target>1.8</maven.compiler.target>
+    <maven.compiler.source>1.11</maven.compiler.source>
```

Comment:
I think you can specify `11` only instead of `1.11`.

Reply:
I'm using Java 1.8 instead of Java 11.